### ISSUE TITLE
iOS VoiceOver does not support aria-roledescription

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#200]
Found in https://github.com/phetsims/trig-tour/issues/129. It doesn't work in Chrome or Safari.

With examples like these, I do not hear the roledescription.

```html
  <button
      tabindex="1"
      role="button"
      aria-label="Point on circle"
      aria-roledescription="movable"
  >Point on circle
  </button>
```

From https://www.digitala11y.com/aria-roledescriptionproperties/.

```html
<div role="region" aria-roledescription="slide" id="slide42" aria-labelledby="slide42heading">
  <h1 id="slide42heading">Quarterly Report</h1><!-- remaining slide contents -->
</div>
```
Lets submit a bug report.

**Comment by jessegreenberg** [comment]
Report submitted using the feedback assistant.

**Comment by jessegreenberg** [comment]
I also submitted a bug report using the webkit bug reporter.
